dspy/teleprompt/gepa/gepa.py

```diff
@@ -516,6 +518,7 @@ def feedback_fn(
             use_wandb=self.use_wandb,
             wandb_api_key=self.wandb_api_key,
             wandb_init_kwargs=self.wandb_init_kwargs,
+            use_mlflow=self.use_mlflow,
```

Comment:
Don't we need to also pass the `- mlflow_tracking_uri: The tracking URI to use for MLflow.
    - mlflow_experiment_name: The experiment name to use for MLflow.`?

Reply:
It just works as expected. The metric is logged to the tracking uri + experiment name globally configured.